### ISSUE TITLE
Improve Accessibility

### ISSUE THREAD

**Issue by xogium** [freescout-helpdesk/freescout#655]
Opening this hopefully easy to fix issue to signal that practically every button in the text editor is unlabled, save the send button. This is not good for blind people  like me, who try to use FreeScout only to not be able to determine which button does what. This is very noticeable when opening a new conversation or trying to reply to one that was already opened. Also the place where you write your message is not visible to a screen reader, even though it can be focused with the tab key.

I think some pretty important WAIAria roles are missing here, and I hope someone will consider improving the accessibility of FreeScout to disabled people, otherwise I will have to find another software, and I really don't want this.

**Comment by freescout-helpdesk** [comment]
Closing in favor of https://github.com/freescout-helpdesk/freescout/issues/1150
